Framework/Algorithms/src/ConvertEmptyToTof.cpp

```diff
@@ -509,20 +508,16 @@ void ConvertEmptyToTof::setTofInWS(const std::vector<double> &tofAxis,
                 << numberOfSpectra << '\n';
 
   auto axisPtr = Kernel::make_cow<HistogramData::HistogramX>(tofAxis);
+  HistogramData::BinEdges edges(tofAxis);
   Progress prog(this, 0.0, 0.2, numberOfSpectra);
-  PARALLEL_FOR2(m_inputWS, outputWS)
```

Comment:
Good :+1:, threading is probably pointless since we just manipulate a few pointers. Personally I would also remove the progress reporting from inside the loop, all it will do is slow things down.